Thoughtless-Eels/Bangazon-WebApp
# Cas branch2

## Link to Ticket
Ticket: # [Link]()

## Description of Proposed Changes

- Created:
a button to pull up the "create a payment page"
enter in the payment information
submit the payment
can edit or delete the payment



## Steps to Test

check that the add payment button works
add a payment
submit the payment
check that the payment exsists
delete or edit the payment
Outline the steps to test
Outline the steps to test

```sh
git fetch --all
git checkout CASBranch2

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
